test/e2e/Makefile

```diff
@@ -126,7 +126,7 @@ endif
 
 .PHONY: run
 run: $(GINKGO) cluster-templates test-extension-deployment ## Run the end-to-end tests
-	$(GINKGO) -v -trace -tags=e2e -focus="$(GINKGO_FOCUS)" $(_SKIP_ARGS) -nodes=$(GINKGO_NODES) --noColor=$(GINKGO_NOCOLOR) $(GINKGO_ARGS) . -- \
+	$(GINKGO) -v --trace --tags=e2e --focus="$(GINKGO_FOCUS)" $(_SKIP_ARGS) --nodes=$(GINKGO_NODES) --no-color=$(GINKGO_NOCOLOR) --junit-report="$(ARTIFACTS)/junit.e2e_suite.01.xml" $(GINKGO_ARGS) . -- \
```

Comment:
Q: The 01 in `junit.e2e_suite.01.xml` previously was the number of the node, and we got more files, one for each node written in parallel. How does this works now?

Reply:
Apparently `--output-dir` is effectively required when using `--junit-report`, since the latter doesn't handle absolute paths. I'm not the only one who found this [confusing](https://github.com/onsi/ginkgo/issues/1001).